src/aim/api/server.py

```
@@ -6,9 +6,14 @@
 """
 
 import asyncio
+import time
+from functools import wraps
```

Ruff (0.12.2)

`functools.wraps` imported but unused

Remove unused import: `functools.wraps`

(F401)